src/core/getFarcasterUserAddresses.test.ts

```diff
@@ -0,0 +1,90 @@
+import { neynarGetCustodyAddressesForFid } from '../utils/neynar/frame/neynarGetCustodyAddressForFid';
```

Comment:
I will say, let's start creating the `src/farcaster` folder, so we can play with the namespaces idea we talk about

Reply:
makes sense! updated this